### ISSUE TITLE
NPE on istack reload

### ISSUE THREAD

**Issue by SlimeDog** [dniym/IllegalStack#51]
Spigot 1.15.2
IllegalStack 2.1.2d

Configuration loads without error on start-up. With no change to configuration:
```
>istack reload
[06:49:13] [Server thread/INFO]: [IllegalStack] - All configuration settings have been reloaded.
[06:49:13] [Server thread/WARN]: Unexpected exception while parsing console command "istack reload"
org.bukkit.command.CommandException: Unhandled exception executing command 'istack' in plugin IllegalStack v2.1.2d
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:47) ~[spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at org.bukkit.command.SimpleCommandMap.dispatch(SimpleCommandMap.java:149) ~[spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchCommand(CraftServer.java:723) ~[spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at org.bukkit.craftbukkit.v1_15_R1.CraftServer.dispatchServerCommand(CraftServer.java:708) [spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at net.minecraft.server.v1_15_R1.DedicatedServer.handleCommandQueue(DedicatedServer.java:443) [spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at net.minecraft.server.v1_15_R1.DedicatedServer.b(DedicatedServer.java:407) [spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at net.minecraft.server.v1_15_R1.MinecraftServer.a(MinecraftServer.java:984) [spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at net.minecraft.server.v1_15_R1.MinecraftServer.run(MinecraftServer.java:824) [spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        at java.lang.Thread.run(Thread.java:834) [?:?]
Caused by: java.lang.NullPointerException
        at enums.Protections.update(Protections.java:760) ~[?:?]
        at main.IllegalStack.loadConfig(IllegalStack.java:385) ~[?:?]
        at main.IllegalStack.ReloadConfig(IllegalStack.java:587) ~[?:?]
        at commands.iStackCommand.onCommand(iStackCommand.java:32) ~[?:?]
        at org.bukkit.command.PluginCommand.execute(PluginCommand.java:45) ~[spigot-1.15.2.jar-2020-06-03-0550:git-Spigot-2f5d615-8637ec0]
        ... 8 more
```

**Comment by dniym** [comment]
Know issue, as a temporary work around please use the in game config editor /istack prot

Fix inbound soon

**Comment by dniym** [comment]
Fixed in Version 2.1.2e

**Comment by SlimeDog** [comment]
True that. Thanks.
